.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,5 +2,16 @@
 
 *Description of changes:*
 
+----
+*Testing*  
+- [ ] Did you run our unit tests? `make test`
+- [ ] Did you run our integration tests? `make integ-test`
```

Comment:
I would prefer to list checks as affirmative statements than questions:

* `make test` passes
* `make integ-test` passes
* Unit tests added for new functionality
* etc...

Reply:
Sounds good this seems more Teutonic, I had taken my template from AppMesh. Updated.